.github/workflows/ci-documentbot.yml

```diff
@@ -29,15 +29,14 @@ on:
 
 jobs:
   label:
-    if: ${{ github.repository == 'apache/pulsar' }}
```

Comment:
this shouldn't be removed

Reply:
@lhotari Thanks for your review and great care!

I've reverted this removal, which was introduced during my test.